code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -43,6 +43,14 @@
 		user << "<span class='notice'>\The [src] is empty!</span>"
 		return
 
+	if(!facidallowed)
+		if(reagents.has_reagent("facid"))
+			qdel(src)
+			var/mob/living/carbon/human/M = user
+			M.adjustBruteLoss(50) //This is high because actually spraying yourself with facid will do around 100 brute, this is supposed to represent a fraction of acid.
+			M.visible_message("<span class='danger'>[user] sprays acid from his [src]! It melts the pump covering him in acid!</span>", \
+			"<span class='userdanger'>The pump of [src] melts as you spray acid from it making it unusuable! and covering you in acid!</span>")

```

Comment:
>The pump of [src] melts as you spray acid from it making it unusuable! and covering you in acid!

Fix the spelling of unusable and get that exclamation point off of it. Maybe put a comma just before "making" so the sentence sounds more fluid.